bin/v2/postgres-setup.sh

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env bash
+# set -ex
+
+NAMESPACE='${NAMESPACE}'
+ENVIRONMENT='${ENVIRONMENT}'
+NAMESPACE_NAME="$1"
+APP_NAME="$2"
+APPS_DIR="../../apps/"
+CLUSTERS_DIR="../../clusters"
+
+PASSWORD=$(openssl rand -base64 15)
+
+function usage() {
+  echo 'usage: ./postgres-setup.sh <namespace> <app_name>'
+}
+
+if [ -z "${NAMESPACE_NAME}" ] || [ -z "${APP_NAME}" ]; then
+  usage
+  exit 1
+fi
+
+if [ ! -f "apps/$NAMESPACE_NAME/preview/aso" ]; then
+  echo "Creating aso"
+  mkdir -p apps/$NAMESPACE_NAME
+  mkdir -p apps/$NAMESPACE_NAME/preview
+  mkdir -p apps/$NAMESPACE_NAME/preview/aso
+
+  (
+    cat <<EOF
+apiVersion: dbforpostgresql.azure.com/v1api20210601
+kind: FlexibleServersConfiguration
+metadata:
+  name: extensions
+  namespace: ${NAMESPACE}
+  annotations:
+    serviceoperator.azure.com/reconcile-policy: detach-on-delete
+spec:
+  owner:
+    name: ${NAMESPACE}-${ENVIRONMENT}
+  azureName: azure.extensions
+  source: user-override
+  value: "btree_gin"
+EOF
+  ) >"apps/$NAMESPACE_NAME/preview/aso/$NAMESPACE_NAME-postgres-config.yaml"
+
+  (
+    cat <<EOF
+apiVersion: dbforpostgresql.azure.com/v1api20210601
+kind: FlexibleServer
+metadata:
+  name: ${NAMESPACE}-${ENVIRONMENT}
+  namespace: ${NAMESPACE}
+spec:
+  version: "14"
```

Comment:
why only version 14?

Reply:
would version 16 be more suitable @thomast1906?